CHANGES.rst

```diff
@@ -257,7 +257,7 @@ Here you can see the full list of changes between each SQLAlchemy-Continuum rele
 
 - Added new plugin architecture
 - Added ActivityPlugin
-- Naming conventions change: History -> Version (to be consistent throughout Continuum)
+- Naming conventions change: History -> Version (to be consistent throughout History)
```

Comment:
We shouldn't chanhe exisiting change log
Maybe we should delete the older change log later